tests/test_camoufox_wardnet_compose.py

```
@@ -0,0 +1,100 @@
+    result = subprocess.run(
+        [
+            "docker",
+            "compose",
+            "-f",
+            "compose.yaml",
+            "-f",
+            "compose.camoufox-wardnet.yaml",
+            "config",
+            "--format",
+            "json",
+        ],
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)